launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -134,13 +139,13 @@ def _load_in_sequence(
         :param composable_node_descriptions: descriptions of composable nodes to be loaded
```

Comment:
Comment not updated yet. "composable_node_descriptions" should be updated to "load_node_requests"

Reply:
Good catch. The same issue is on `master`. See https://github.com/ros2/launch_ros/pull/207 for a fix. I'll incorporate together in this PR after #207 is merged.

Reply:
Fixed in b3159a5